lldb/test/Shell/SwiftREPL/Dict.test

```diff
@@ -7,19 +7,22 @@
 // once per element.
 
 [1:2, 2:3, 3:9]
-// DICT-LABEL: $R0: [Int : Int] = 3 key/value pairs {
+// DICT-LABEL: $R0: [Int : Int] = 
```

Comment:
I'm just curious: what does the full new output look like? Is the `3 key/value pairs` on a different line now?

Reply:
Yes, the new output looks like this: 
```
x: [Foundation.URL : Int] = {
  _variant = {
    object = {
      rawValue = 2 key/value pairs {
        [0] = {
          key = "https://github.com"
          value = 4
        }
        [1] = {
          key = "https://apple.com"
          value = 23
        }
      }
    }
  }
}
```

Reply:
Shouldn't the `_variant = ` be hidden by the Dictionary data formatter? Is the formatter not triggering?

Reply:
@adrian-prantl an update on this: the changed isn't caused by this patch, but by implementing `TypeSystemSwiftTypeRef::GetFullyUnqualifiedType`, which this was in preparation for. I've removed the test changes and will look into why a typeref type ins't triggering the formatter.

Reply:
Thanks for investigating!